src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -288,4 +285,49 @@ private function determineSymfonyState()
 
         return $versionState;
     }
+
+    /**
+     * Gets the names, paths and installed versions of the bundles enabled in
+     * the application.
+     *
+     * @return array
+     */
+    private function getBundleData()
+    {
+        $bundles = array();
+
+        $composerLockPath = $this->kernel->getRootDir().'/../composer.lock';
+        if (!file_exists($composerLockPath)) {
+            $installedPackages = array();
+        } else {
+            $composerLock = json_decode(file_get_contents($composerLockPath), true);
+            foreach (array_merge($composerLock['packages'], $composerLock['packages-dev']) as $package) {
+                $installedPackages[$package['name']] = $package['version'];
+            }
+        }
+
+        foreach ($this->kernel->getBundles() as $name => $bundle) {
+            $bundlePath = $bundle->getPath();
+
+            $bundleVersion = null;
+            foreach ($installedPackages as $packageName => $packageVersion) {
+                if (strpos($bundlePath, '/vendor/'.$packageName)) {
```

Comment:
add a ``/`` after the package name, to avoid issues with overlapping package names (``knplabs/knp-menu`` vs ``knplabs/knp-menu-bundle`` for instance, where your code would use the ``knplabs/knp-menu`` version as it appears first in the lock file and would match your condition)